File: week01/BestTimeToBuyAndSellStockII.java
public int maxProfit(int[] prices) {
        int totalProfit = 0;
        
        for(int i = 0; i < prices.length -1; i++)
            if(prices[i] < prices[i+1]) totalProfit = totalProfit + (prices[i+1] - prices[i]);
        
        return totalProfit;
    }

File: week01/bestTimeToBuyAndSellStockII.cpp
#include "bits/stdc++.h"
using namespace std;
int maxProfit(vector<int>& prices) {
	int mx = 0;
	for(int i = 0; i + 1 < prices.size(); i++)
	{
		if(prices[i + 1] < prices[i])
		{
			mx += prices[i + 1] - prices[i];
		}
	}
	return mx;
}
